sky/clouds/oci.py

```diff
@@ -256,6 +256,8 @@ def make_deploy_resources_variables(
             'region': region.name,
             'cpus': cpus,
             'memory': resources.memory,
+            'disk_size': resources.disk_size,
+            'vpu': f'{oci_conf.BOOT_VOLUME_VPU[resources.disk_tier]}',
```

Comment:
For all other clouds, we set the default to the `medium` tier. Can we do the similar thing for OCI?
https://github.com/skypilot-org/skypilot/blob/master/sky/clouds/gcp.py#L705-L713

Reply:
For OCI, balanced performance is the default and fit to most cases (I think it would means medium in other clouds).  My understanding is for computation tasks in sky the price for disk performance is piece of cake comparing to GPU cost esp. when mostly tasks would be hours/days/weeks work instead of months, right? If my understanding is correct, then we would set the default to 'Higher performnace' if neccessary. @Michaelvll

Reply:
Currently, we have no support for high-performance disks in Azure (high-performance disks such as UltraDisk cannot be used as boot disks in Azure). If we set the default disk tier to 'high', we might not be able to ensure similar default disk speeds across different clouds.